discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -174,9 +190,9 @@ def set_olm_operator(self, operator_name, properties=None):
 
         self.api_client.update_cluster(self.id, {'olm_operators': olm_operators})
 
-    def set_host_roles(self, requested_roles=None):
+    def set_host_roles(self, num_masters, num_workers, requested_roles=None):
```

Comment:
default for num_masters and num_workers should be taken from _config

Reply:
num_workers is not part of cluster configuration

Reply:
I'm adding num_masters temporarily to Config and probably will be moved to Nodes in MGMT-5370